Session/Calls/Call Management/SessionCallManager+CXCallController.swift

```diff
@@ -16,7 +16,8 @@ extension SessionCallManager {
         reportOutgoingCall(call)
         
         if callController != nil {
-            let handle = CXHandle(type: .generic, value: call.sessionId)
+            // Show contact name opening outgoing call in apple watch
+            let handle = CXHandle(type: .generic, value: call.contactName)
```

Comment:
It looks like we create a `CXHandle` in `SessionCallManager.reportIncomingCall` as well - can you please update that to match?